### ISSUE TITLE
3.1.* breaks the `service_clients.request_access_key` method

### ISSUE THREAD

**Issue by ramon-prieto** [Authress/authress-sdk.py#41]
```
return self.client.service_clients.request_access_key(client_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/decorator.py", line 40, in wrapper_function
    return vd.call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/decorator.py", line 134, in call
    return self.execute(m)
           ^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/decorator.py", line 206, in execute
    return self.raw_function(**d, **var_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/api/service_clients_api.py", line 792, in request_access_key
    return self.request_access_key_with_http_info(client_id, **kwargs)  # noqa: E501
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/decorator.py", line 40, in wrapper_function
    return vd.call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/decorator.py", line 134, in call
    return self.execute(m)
           ^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/decorator.py", line 206, in execute
    return self.raw_function(**d, **var_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/api/service_clients_api.py", line 890, in request_access_key_with_http_info
    return self.api_client.call_api(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/http_client.py", line 386, in call_api
    return self.__call_api(resource_path, method,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/http_client.py", line 224, in __call_api
    return_data = self.deserialize(response_data, response_type)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/http_client.py", line 296, in deserialize
    return self.__deserialize(data, response_type)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/http_client.py", line 335, in __deserialize
    return self.__deserialize_model(data, klass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/http_client.py", line 762, in __deserialize_model
    return klass.from_dict(data)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/authress/models/client_access_key.py", line 95, in from_dict
    _obj = ClientAccessKey.parse_obj({
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/main.py", line 526, in parse_obj
    return cls(**obj)
           ^^^^^^^^^^
  File "/Users/ramonprieto/src/AIBoostrapper/trially/llm-agents-service/.venv/lib/python3.12/site-packages/pydantic/v1/main.py", line 341, in __init__
    raise validation_error
pydantic.v1.error_wrappers.ValidationError: 1 validation error for ClientAccessKey
client_id
  extra fields not permitted (type=value_error.extra)
```

**Comment by bruvduroiu** [comment]
Seeing same issue after upgrading from `3.0.96` to `3.1.106`

**Comment by wparad** [comment]
Thanks for reporting this. We actually aren't really happy with pydantic due to the issues it is causing in our SDK. We are going to release an update to the 3.1 version to remove the problem, and strategize on an alternative way to handle this going forward.

**Comment by wparad** [comment]
This is fixed in `3.1.109`
